lib/utils/pollingUtil.js

```diff
@@ -75,12 +76,15 @@ class SftpPolling extends PollingTrigger {
 
   // eslint-disable-next-line class-methods-use-this
   prepareMessageDescription(file) {
+    const dir = getDirectory(this.cfg);
```

Comment:
```suggestion
    const dir = path.posix.dirname(filename);;
```
currently we dont use directiry in cfg